apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -24,9 +25,11 @@ export class Nav extends React.Component<INavProps, INavState> {
       : null;
 
     return (
-      <nav className={ styles.nav } role='navigation'>
-        { links }
-      </nav>
+      <FocusZone ref='focusZone' direction={ FocusZoneDirection.vertical } role='menubar' >
```

Comment:
Internal nav is already assigned `navigation` (its redundant as well `<nav>`s are implicit `navigation`) and focus zone will auto focus the first active element which would be one of the list options. IMO, you won't require any role on FocusZone.